components/scrapd-home-section/component.js

```diff
@@ -22,7 +22,8 @@ const ScrapdHomeSection = props => {
       backgroundColor: 'white',
       fontSize: ['2em', '2.75em'],
       fontWeight: 'bold',
-      color: '#003CFF'
+      color: '#003CFF',
+      padding: '0px 15px'
```

Comment:
Added some padding to the left and right sides. Here's how it looked before:
<img width="399" alt="Screen Shot 2020-01-28 at 8 00 04 PM" src="https://user-images.githubusercontent.com/48178951/73326915-7a330100-4219-11ea-86a0-751faa14c93e.png">